src/pages/iou/MoneyRequestModal.js

```diff
@@ -329,6 +329,11 @@ const MoneyRequestModal = (props) => {
             );
             return;
         }
+        console.debug(selectedParticipants);
+        if (!selectedParticipants[0].login) {
+            // TODO - request to the policy expense chat. Not implemented yet!
```

Comment:
Do we have another issue for this or it should be covered by this PR?

Reply:
Pretty sure we'll handle this separately, once the API has been updated to accept workspaces.

Reply:
done!